ChromeDevToolsClient/src/main/java/com/hubspot/chrome/devtools/client/ChromeDevToolsClient.java

```diff
@@ -128,12 +128,10 @@ private TargetID startNewTarget(String host, int port) {
       throw new ChromeDevToolsException("Unable to find available chrome session info.");
     }
 
-    List<ChromeSessionInfo> sessions = response.getAs(new TypeReference<>() {});
-    if (sessions.size() == 0) {
-      throw new ChromeDevToolsException("Unable to create new websocket target");
-    }
+    ChromeSessionInfo newSession = response.getAs(new TypeReference<>() {});
+    LOG.debug("new session: {}", newSession);
```

Comment:
This seems like a more major than expected change that has no way of being backwards compatible. Or was this simply a broken code path before that was unable to be hit because everything had an initial target by default?

Reply:
I believe this was a broken code path, going off the description of the `/json/new` endpoint. Think this call always returned a single new session.

> Opens a new tab. Responds with the websocket target data for the new tab.

Reply:
👍